welcome/index.adoc

```diff
@@ -208,8 +211,11 @@ xref:../applications/working_with_helm_charts/understanding-helm.adoc#understand
 xref:../operators/operator_sdk/helm/osdk-helm-support.adoc#osdk-helm-support[Helm], or configure xref:../operators/operator_sdk/osdk-monitoring-prometheus.adoc#osdk-monitoring-prometheus[built-in Prometheus monitoring] by using the Operator SDK.
 
 - **Reference the xref:../rest_api/index.adoc#api-index[REST API index]**: Learn about {product-title} application programming interface endpoints.
-endif::[]
-endif::[]
+
+// Need to provide a link closer to 4.15 GA
+- **Software Supply Chain Enhancements**: The PipelineRun *Details* view in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of PipelineRuns within a Project. Learn about the Software Supply Chain enhancements and how to deploy them.
```

Comment:
- "PipelineRun" looks like an object of some type. Probably needs backticks and some noun to follow it. 
- Should also double-check "Software Supply Chain" capitalization. Is it a component, or should that just be normal lower case use?

```suggestion
- **Software Supply Chain enhancements**: The `PipelineRun` *Details* view in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of `PipelineRun` objects within a project. Learn about the Software Supply Chain enhancements and how to deploy them.
```

Reply:
https://www.redhat.com/en/about/press-releases/red-hat-introduces-red-hat-trusted-software-supply-chain

Reply:
"Red Hat Trusted Software Supply Chain" then I guess? What a mouthful :dizzy_face:

Reply:
Sorry, I also pinged Gaurav Trivedi on Slack to confirm :pen:

Reply:
PR https://github.com/openshift/openshift-docs/pull/70231 is planned for merging before the 4.15 GA. I'll create a subtask on my Welcome page Jira to comment in the link when the SSCS doc is published.

Reply:
For "details page", see [link](https://docs.openshift.com/pipelines/1.13/create/working-with-pipelines-web-console.html#op-creating-pipeline-templates-admin-console_working-with-pipelines-web-console).